### DISCUSSION TITLE
"publicPath" only works for workers and not for other generated assets

### DISCUSSION THREAD

**Discussion by enzocisse** [microsoft/monaco-editor#3770] [Q&A]
Specifying `publicPath` in the webpack config only works for the workers.
Other assets like the ttf font (for icons), are served by the default `process.env.ASSET_PATH`.

**Comment by alexdima** [comment]
@enzocisse AFAIK we aren't doing anything about the `ttf` font inside this plugin, in our samples we use something like the following:
```
  {
     test: /\.ttf$/,
     use: ['file-loader']
  }
```

So I believe packaging/loading the `ttf` font is delegated to that file loader, which does not receive any options given to the plugin. This is similar to the `css`, which is not handled explicitly by this plugin.

**Comment by enzocisse** [comment]
Thanks for the response and the great work you are doing!
I think It would be nice if the plugin could output all assets generated by monaco to `publicPath`. Doing so will probably reduce the friction to using monaco as this plugin becomes its only configuration entry point.
